### PR TITLE
Fixes #12634: .pyc file are not cleand up during postinst

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/12634

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: ncf-api-virtualenv/SPECS/ncf-api-virtualenv.spec [hunk 1]
```diff
@@ -247,6 +247,10 @@ fi
 # Enable mod_wsgi using a2enmod
 a2enmod wsgi >/dev/null 2>&1
 
+# Remove .pyc files to ensure we don't end up with outdated files
+rm -f %{installdir}/tools/ncf.pyc
+rm -f %{installdir}/tools/ncf_constraints.pyc
```
Comment (VinceMacBuche) [inline ncf-api-virtualenv/SPECS/ncf-api-virtualenv.spec]: you should also rm ncf_rudder.pyc

Comment (VinceMacBuche) [reply ncf-api-virtualenv/SPECS/ncf-api-virtualenv.spec]: (in fact i would do 

> find %{installdir}/tools -name "*.pyc" -exec rm -f {} \\;

Comment (VinceMacBuche) [reply ncf-api-virtualenv/SPECS/ncf-api-virtualenv.spec]: Hmm, in fact ncf_rudder does not make a pyc ..., i would still do a find :)

**Comment by peckpeck** [comment]
We should create another issue to make sure we never forget a .pyc file in any packages.
For today, this is sufficient I think

**Pull request merged**